packages/dd-trace/src/plugins/tracing.js

```diff
@@ -22,6 +26,14 @@ class TracingPlugin extends Plugin {
     this.addTraceSub('finish', message => {
```

Comment:
Should this be explicitly deprecated?

Reply:
Well, this isn't in the public API so I'm not sure what deprecated would mean here. If you mean removed, then yes, once there are no plugins publishing it. For now, it makes sense to leave it as all plugins except DNS need it.

Reply:
I just meant to add some sort of TODO.